core/src/apps/management/sd_backup.py

```diff
@@ -0,0 +1,30 @@
+from trezor.wire import ProcessError
+from trezor import wire
+
+def is_sdbackup_present():
+    from trezor import sdcard
+    return sdcard.is_present()
+
+
+
+async def sd_card_backup_seed(
+    ctx: wire.Context, mnemonic_secret: bytes
+) -> None:
+    from storage.sd_seed_backup import set_sd_seed_backup
+    from apps.common.sdcard import ensure_backup_sd_card
+
+    # Ensure sd card for backup
+    await ensure_backup_sd_card(ctx)
+
+    # Write seed backup
+    set_sd_seed_backup(mnemonic_secret)
+
+
+def verify_sd_backup_seed(mnemonic_secret: bytes) -> None:
```

Comment:
we will want to use this function in the "dry-run recovery" too in the next iteration, so perhaps it would be better as `-> bool` ?

Reply:
Done in 2e71232f57f6a1ac4119959677f12e193b4024d3